readthedocs/builds/models.py

```diff
@@ -168,7 +169,8 @@ def identifier_friendly(self):
         return self.identifier
 
     def get_subdomain_url(self):
-        return resolve(project=self.project, version_slug=self.slug)
+        private = self.privacy_level == 'private'
```

Comment:
This should use a constant for `private`, or maybe even just move this check to a `is_private` method on the build model.